.github/container/manifest.yaml

```diff
@@ -96,9 +96,12 @@ pathwaysutils:
   latest_verified_commit: 359776d454940ffaa337c36d1df16308d44a95a9
   mode: pip-vcs
 axlearn:
-  url: https://github.com/Steboss/axlearn.git
+  url: https://github.com/apple/axlearn.git
+  mirror_url: https://github.com/nvjax-svc-0/axlearn.git
```

Comment:
We're not actually using this, right? https://github.com/apple/axlearn/pull/1339 is from your personal fork.